tests/release/release.go

```diff
@@ -0,0 +1,164 @@
+package release
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	ecp "github.com/hacbs-contract/enterprise-contract-controller/api/v1alpha1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	applicationapiv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	klog "k8s.io/klog/v2"
+	"knative.dev/pkg/apis"
+)
+
+var snapshotComponents = []applicationapiv1alpha1.SnapshotComponent{
+	{Name: "component-1", ContainerImage: "quay.io/redhat-appstudio/component1@sha256:d5e85e49c89df42b221d972f5b96c6507a8124717a6e42e83fd3caae1031d514"},
+	{Name: "component-2", ContainerImage: "quay.io/redhat-appstudio/component2@sha256:a01dfd18cf8ca8b68770b09a9b6af0fd7c6d1f8644c7ab97f0e06c34dfc5860e"},
+	{Name: "component-3", ContainerImage: "quay.io/redhat-appstudio/component3@sha256:d90a0a33e4c5a1daf5877f8dd989a570bfae4f94211a8143599245e503775b1f"},
+}
+
+var ecPolicy = ecp.EnterpriseContractPolicySpec{
+	Description: "Red Hat's enterprise requirements",
+	Sources: []string{
+		"https://github.com/hacbs-contract/ec-policies",
+	},
+	Exceptions: &ecp.EnterpriseContractPolicyExceptions{
+		NonBlocking: []string{"tasks", "attestation_task_bundle", "java", "test", "not_useful"},
+	},
+}
+
+var paramsReleaseStrategy = []appstudiov1alpha1.Params{}
+
+var _ = framework.ReleaseSuiteDescribe("test-release-service-happy-path", Label("release", "HACBS"), func() {
```

Comment:
Please add JIRA issue key in description.

Reply:
@johnbieren the reason we add the Jira is that in e2e-tests to make it much easier users can follow what is this PR about .

Reply:
@jkopriva  , Done

Reply:
It makes sense to me in the commit message. It does not make sense to me in the test name. So when this test breaks and we have to refactor it sometime we just rename the test again? I don't see the point